h2o-py/h2o/model/model_reliance.py

```diff
@@ -0,0 +1,25 @@
+# -*- encoding: utf-8 -*-
+"""
+H2O Model Reliance.
+
+:copyright: (c) 2020 H2O.ai
+:license:   Apache License Version 2.0 (see LICENSE for details)
+"""
+
+from h2o.frame import H2OFrame
+from h2o.expr import ExprNode
+
+
+def model_reliance(frame, model):
```

Comment:
Why model_reliance?

Reply:
Got motivated by literature I was reading. I will rename it something along Permutation Feature Importance